testplan/exporters/testing/http/__init__.py

```diff
@@ -24,8 +24,8 @@ class HTTPExporterConfig(ExporterConfig):
     @classmethod
     def get_options(cls):
         return {
-            ConfigOption('url', default=None): Or(None,
```

Comment:
Do you need to make similar changes to the Config options for the other exporters?

Reply:
Other exporters already changed 2 months ago. simply like **ConfigOption('pdf_path'): str**